src/doc/trpl/structs.md

```diff
@@ -87,3 +87,32 @@ fn main() {
     point.y = 6; // this causes an error
 }
 ```
+
+# Update syntax
+
+A `struct` can include `..` to indicate that you want to update just some
+values of a struct. For example:
```

Comment:
Semantic here is to create a copy of a value and update zero or more of the fields to a different value at the same time. 

You are not restricted to mutating the same value (i.e. you may assign it to a different variable), and you can use `let another_point = Point3d { ..point }` as a way to copy and not move a value (e.g. all fields implement copy, but Point3d does not).